HPXMLtoOpenStudio/resources/battery.rb

```diff
@@ -1,17 +1,21 @@
 # frozen_string_literal: true
 
-# TODO
+# Collection of methods for adding battery-related OpenStudio objects.
 module Battery
-  # TODO
+  # Apply a home battery to the model using OpenStudio ElectricLoadCenterStorageLiIonNMCBattery, ElectricLoadCenterDistribution, ElectricLoadCenterStorageConverter, OtherEquipment, and EMS objects.
+  # Battery without PV specified, and no charging/discharging schedule provided; battery is assumed to operate as backup and will not be modeled.
+  # The system may be shared, in which case nominal/usable capacity (kWh) and usable fraction are apportioned to the dwelling unit by total number of bedrooms served.
+  # A battery may share an ElectricLoadCenterDistribution object with PV; electric buss type and storage operation scheme are therefore changed.
+  # Round trip efficiency is (temporarily) applied as an EMS program b/c E+ input is not hooked up.
   #
   # @param runner [OpenStudio::Measure::OSRunner] Object typically used to display warnings
   # @param model [OpenStudio::Model::Model] OpenStudio Model object
   # @param nbeds [Integer] Number of bedrooms in the dwelling unit
-  # @param pv_systems [TODO] TODO
-  # @param battery [TODO] TODO
+  # @param pv_systems [HPXML::PVSystems] Object that defines each solar electric photovoltaic (PV) system
+  # @param battery [HPXML::Battery] Object that defines a single home battery
   # @param schedules_file [SchedulesFile] SchedulesFile wrapper class instance of detailed schedule files
   # @param unit_multiplier [Integer] Number of similar dwelling units
-  # @return [TODO] TODO
+  # @return [nil or void] nil if unscheduled battery w/out PV; battery is not modeled
```

Comment:
I know we found a source that recommends using void, but the difference between nil and void is awfully nuanced. I don't think there's any real practical difference. Maybe we should just go with `nil` everywhere and keep it simple?